simple/util/dc_client.py

```diff
@@ -244,6 +244,49 @@ def resolve_entity_type(entity_dcids: list[str]) -> str:
   return common_entity_types.pop() if common_entity_types else ""
 
 
+def get_property_of_entities(entities: list[str],
+                             property_name: str) -> dict[str, str]:
+  return asyncio.run(get_property_of_entities_async(entities, property_name))
+
+
+async def get_property_of_entities_async(entities: list[str],
+                                         property_name: str) -> dict[str, str]:
+
+  chunks = chunked(entities, _BATCH_SIZE)
+
+  result: dict[str, str] = {}
+  async with AsyncClient(limits=_HTTPX_LIMITS, timeout=None) as client:
+    futures: dict[str, str] = [
+        _get_property_of_entities_chunk(client, chunk, property_name)
+        for chunk in chunks
+    ]
+    for result_chunk in await asyncio.gather(*futures):
+      result.update(result_chunk)
+
+  return result
+
+
+async def _get_property_of_entities_chunk(client: AsyncClient,
+                                          entities_chunk: list[str],
+                                          property_name: str) -> dict[str, str]:
+  data = {
+      "nodes": entities_chunk,
+      "property": f"->{property_name}",
+  }
+
+  logging.info("Fetching nodes: %s", data)
+  response = await post_async(client, path="/v2/node", data=data)
+
+  result_chunk: dict[str, str] = {}
+  for entity_dcid, entity_data in response.get("data", {}).items():
```

Comment:
maybe add a note to handle pagination later..

Reply:
Done.